torchtitan/models/deepseek_v3/README.md

```diff
@@ -12,8 +12,8 @@ python scripts/download_tokenizer.py --repo_id deepseek-ai/DeepSeek-V3
 ```
 
 ```bash
-# For 16B model support:
-python scripts/download_tokenizer.py --repo_id deepseek-ai/deepseek-moe-16b-chat
+# DeepSeek 16B tokenizer:
+python scripts/download_tokenizer.py --repo_id deepseek-ai/deepseek-moe-16b-base
 ```
```

Comment:
Thanks for your change! We should add explicit comments/Notes here, saying we are reusing the tokenzier  from deepseek-ai/deepseek-moe-16b-base, to help user test and run 16B model. And this is not official tokenizer for deepseek-v3-16B model. But our Deepseek-v3 model has different architecture from the deepseek-moe-16b (Attention implementaiton, MoE router implementation), and it's not feasible to load deepseek-moe-16b model weights into deepseek-v3-16B